en/docs/edit-the-code/notebooks.md

```diff
@@ -0,0 +1,36 @@
+# Notebooks
+
+Ballerina notebooks can be created in VS Code using the `.balnotebook` extension for filename. Markdown and Ballerina code snippets are supported by Ballerina notebook cells.
+
+!!! Info
+    Make sure your VS Code version is `1.67.0` or higher when using Ballerina notebooks.
+
+## Create a Notebook 
+
+The command below creates a notebook with the given filename.
+
+<img src="../img/edit-the-code/notebooks/notebook-create.gif" width="100%" />
```

Comment:
Use the standard image formatting format with CSS styles. i.e. use `{.cInlineImage-half}` and `{.cInlineImage-full}` appopriately

Reply:
Updated.

Reply:
Done.